src/main/java/uk/gov/hmcts/reform/civil/enums/ClaimState.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.civil.enums;
+
+public enum ClaimState {
+    TRANSFERRED
```

Comment:
sorry i dont quite get why do we need this claim state. perhaps when you do a demo and explain it will be clear